presentation/slides.md

```
@@ -0,0 +1,525 @@
+  strong { color: #f5c518; }
+  em { color: #9cdcfe; font-style: normal; }
+  table { width: 100%; border-collapse: collapse; }
+  th { background: #1e1e1e; color: #f5c518; padding: 8px 12px; }
+  td { padding: 8px 12px; border-bottom: 1px solid #333; }
+  blockquote { border-left: 4px solid #f5c518; padding-left: 16px; color: #aaa; font-style: italic; }
+---
+
+<!-- Slide 1: Title -->
+# 🏆 RaceForThePrize
+
+### *Browser Performance. Head-to-Head. No Judges.*
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)